Python/PRP/pyCreatable.cpp

```diff
@@ -104,6 +105,33 @@ PY_METHOD_VA(Creatable, write,
     Py_RETURN_NONE;
 }
 
+PY_GETSET_GETTER_DECL(Creatable, prc)
+{
+    hsRAMStream* S = new hsRAMStream();
+    pfPrcHelper* prc = new pfPrcHelper(S);
+    self->fThis->prcWrite(prc);
+
+    S->rewind();
+    size_t dataSize = S->size() - S->pos();
+    unsigned char* buf = new unsigned char[dataSize];
+    S->read(dataSize, buf);
+
+    auto data = ST::string::from_utf8((const char*)buf, dataSize);
+    PyObject* result = PyString_FromSTString(data);
+    delete[] buf;
+
+    return result;
+}
+
+PY_PROPERTY_SETTER_MSG(Creatable, prc, "To set the contents, you must re-create the object")
```

Comment:
Forgot to update the error text, eh? ;)

Reply:
I did update it...

Reply:
Why is this a property?  Creatables have no such member or property -- if anything, this could be a function on plCreatable which is then exposed to PyHSPlasma (also as a function).

Reply:
It's a property because I had originally intended it to function as one, as an interface to prcParse and prcWrite.  I never got around to implementing the setter to cover prcParse, and a script @Hoikas and I are using required my existing getter, so I submitted this as-is with the setter essentially stubbed out.

If it's preferable (due to a lack of interest in setting the object PRC from Python), I could rewrite this as an ordinary function.